src/components/Common/UI/Switch/Switch.module.scss

```diff
@@ -0,0 +1,98 @@
+:global(.react-aria-Switch) {
+  --selected-color: var(--g-colors-gray-1000);
+  --selected-color-pressed: var(--g-colors-gray-900);
+
+  position: relative;
+
+  font-size: toRem(15);
+  color: var(--g-typography-textColor);
+  cursor: pointer;
+  forced-color-adjust: none;
+
+  .body {
+    display: flex;
+    align-items: start;
+    gap: 0.571rem;
+  }
+
+  .switch-details {
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    top: -2px;
+  }
+
+  .indicator {
+    width: 2.286rem;
+    height: 1.143rem;
+    background: var(--g-colors-gray-200);
+    border-radius: 1rem;
+    transition: all 200ms;
+    position: relative;
+
+    &::after {
+      content: '';
+      position: absolute;
+      top: 0.143rem;
+      left: 0.143rem;
+      width: 0.857rem;
+      height: 0.857rem;
+      background: var(--g-colors-gray-600);
+      border-radius: 50%;
+      transition: all 200ms;
+    }
+  }
+
+  &[data-hovered] .indicator {
+    background: var(--g-colors-gray-300);
+  }
+
+  &[data-pressed] .indicator {
+    background: var(--g-colors-gray-400);
+  }
+
+  &[data-focus-visible] .indicator {
+    outline: var(--g-focus-borderWidth) solid var(--g-focus-color);
+    outline-offset: 2px;
+  }
+
+  &[data-selected] {
+    .indicator {
+      background: var(--selected-color);
+
+      &::after {
+        background: var(--g-colors-gray-100);
+        transform: translateX(1.143rem);
+      }
+    }
+
+    &[data-pressed] .indicator {
+      background: var(--selected-color-pressed);
+    }
+  }
+
+  &[data-disabled] {
+    color: var(--g-typography-disabledTextColor);
+    cursor: not-allowed;
+
+    .indicator {
+      background: var(--g-colors-gray-100);
+
+      &::after {
+        background: var(--g-colors-gray-300);
+      }
+    }
+
+    &[data-selected] .indicator::after {
+      background: var(--g-colors-gray-400);
+    }
+
+    & + .react-aria-Switch-description {
+      color: var(--g-typography-disabledTextColor);
+    }
```

Comment:
Any description/label styles we should be able to remove since we updated it to use HorizontalFieldLayout